spec/outputs/graphite_spec.rb

```diff
@@ -1,86 +1,49 @@
 require_relative '../spec_helper'
 
-describe LogStash::Outputs::Graphite do
-
-  let(:port) { 4939 }
-  let(:config) do <<-CONFIG
-     input {
-        generator {
-          message => "foo=fancy bar=42"
-          count => 1
-          type => "generator"
-        }
-      }
-
-      filter {
-        kv { }
-      }
-
-      output {
-        graphite {
-          host => "localhost"
-          port => #{port}
-          metrics => [ "hurray.%{foo}", "%{bar}" ]
-        }
-      }
-  CONFIG
+class LogStash::Outputs::Graphite
```

Comment:
Begin just a handful of lines, creating an extra file (already 3 exist in spec/) starts making this harder to understand instead of making clear. you have to open 4 files to know all things that happen in the spec.
My suggestion is to add this extension to the `spec_helper.rb`